submissions/kolajoTomike/Database/TransportationCompany.sql

```diff
@@ -20,27 +20,50 @@ select * from passengers;
 -- creating tables for Trip Details
 CREATE TABLE trip(
 id INT NOT NULL AUTO_INCREMENT,
-trip_id INT NOT NULL,
-registered_passengers_id INT NOT NULL,
+passengers_id INT NOT NULL DEFAULT(1),
 passenger_class INT NOT NULL,
 passenger_ticket VARCHAR(255) NOT NULL,
 trip_fare decimal(10, 2) not null,
 assigned_cabin VARCHAR(25) NOT NULL,
-parent_or_children_abroad INT(15) NOT NULL,
-siblings_or_spouses_abroad INT (15) NOT NULL,
+parent_or_children_abroad INT NOT NULL,
+siblings_or_spouses_abroad INT NOT NULL,
 embarkation_point VARCHAR (255) NOT NULL,
 primary key (id),
-foreign key (trip_id) references passengers(id)
+foreign key (passengers_id) references passengers(id)
 );
 select * from trip;
+ drop table trip;
 
 CREATE TABLE accidents_cases (
 id INT NOT NULL AUTO_INCREMENT,
-passenger_id INT NOT NULL,
-accident_status TINYINT(1) NOT NULL,
+passenger_id INT NOT NULL DEFAULT(1),
+accident_status TINYINT NOT NULL,
 primary key (id),
 foreign key (passenger_id) references passengers(id)
 );
 
-select * from accidents_cases
+select * from accidents_cases;
+DROP TABLE accidents_cases;
 
+-- Task3 Transportation Data Query--
+
+#1. Total number of passengers who survived 
+select count(*) from accidents_cases where accident_status > 0;
+-- Answer => 342
+
+#2 Total number of unsurvived passengers
+select count(*) from accidents_cases where accident_status = 0;
+-- Answers => 549
+
+#3 Name and sex of passengers under the age 27 that embarked at Queenstown and Cherbourg
+SELECT passengers.full_name, passengers.sex from passengers inner join trip on passengers_id = trip.passengers_id where age < '27' and embarkation_point like "Q%" or embarkation_point like "C%";
```

Comment:
Break down this line of code.

Also, you should specify the `table_name.column_name` for column names that appear on the same tables you are joining. You can't have this `passengers_id = trip.passengers_id `